packages/plugin/src/rules/no-hashtag-description.ts

```diff
@@ -56,16 +55,17 @@ const rule: GraphQLESLintRule = {
     schema: [],
   },
   create(context) {
+    const selector = `${Kind.DOCUMENT}[definitions.0.kind!=/^(${Kind.OPERATION_DEFINITION}|${Kind.FRAGMENT_DEFINITION})$/]`;
```

Comment:
does `definitions.0` will effect files with multiple documents? (for example: a document with query and a fragment)

Reply:
This selector will skip documents where first definition is either operation or fragment